src/HydraAuction/Tx/StandingBid.hs

```diff
@@ -85,28 +100,43 @@ getStadingBidDatum standingBidUtxo =
       _ -> error "Impossible happened: No inline data for standing bid"
     _ -> error "Wrong number of standing bid UTxOs found"
 
-currentWinningBidder :: AuctionTerms -> Runner (Maybe PubKeyHash)
+currentWinningBidder :: (MonadNetworkId m, MonadQueryUtxo m) => AuctionTerms -> m (Maybe PubKeyHash)
 currentWinningBidder terms = do
   standingBidUtxo <- scriptUtxos StandingBid terms
   let StandingBidDatum {standingBidState} = getStadingBidDatum standingBidUtxo
   return $ case standingBid standingBidState of
     (Just (BidTerms {bidBidder})) -> Just bidBidder
     Nothing -> Nothing
 
-getApprovedBidders :: AuctionTerms -> Runner ApprovedBidders
+getApprovedBidders :: (MonadNetworkId m, MonadQueryUtxo m) => AuctionTerms -> m ApprovedBidders
 getApprovedBidders terms = do
   standingBidUtxo <- scriptUtxos StandingBid terms
   let StandingBidDatum {standingBidState} = getStadingBidDatum standingBidUtxo
   pure $ approvedBidders standingBidState
 
 newBid :: AuctionTerms -> Natural -> Runner ()
```

Comment:
Why are some type signatures using the concrete `Runner` type, while others use the `(MonadNetworkId m, MonadQueryUtxo m)` class constraints?

Would it be better to use the class constraints uniformly throughout?